lib/LMSManagers/LMSFinanceManager.php

```diff
@@ -3785,6 +3785,7 @@ public function GetReceiptList(array $params)
 				FROM documents
 				LEFT JOIN numberplans ON (numberplanid = numberplans.id)
 				LEFT JOIN vusers ON (userid = vusers.id)
+				LEFT JOIN receiptcontents ON documents.id = docid AND type = ?
```

Comment:
A tu po co test typu dokumentu? To w receiptcontents może siedzieć zawartość innego dokumentu niż DOC_RECEIPT?